### PR TITLE
Create .editorconfig

### PR DESCRIPTION
**Pull request by clouless**

### CODE REVIEW

**Comment by IonicaBizau** [comment]
Just realized my review was still pending. :sweat_smile:

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+insert_final_newline=true
+trim_trailing_whitespace=true
+
+[*.php]
+indent_style=tab
+indent_size=4
+
+[{*.css, *.scss, *.js, *.html, package.json, *.yml}]
+indent_style=space
+indent_size=2
```
Comment (clouless) [inline .editorconfig]: ok.
the commas thing is not relevant to editorconfig. That should be handled by eslint or similar.
I wrote a [BlogPost about Eslint](https://comsysto.com/blog-post/enforce-javascript-codestyle-for-your-react-application) maybe that helps.

For the indent 4 and php => will do 😄

**Comment by clouless** [comment]
I just checked the code and I see a mixture of 2 space and 4 space indent.

```
module.exports = class Anser {
    static escapeForHtml (txt) {
      return new Anser().escapeForHtml(txt);
    }
```

The method is 4 but the `return new ...` is 2 space.
Same for Unit Test. Maybe you want to fix that yourself 😄

**Comment by IonicaBizau** [comment]
Ah, will take care of that :grin:

**Comment by IonicaBizau** [comment]
@clouless Addressed in #26. Thanks for that! :+1:

**Pull request merged**